utils_linux.go

```diff
@@ -280,29 +291,32 @@ func (r *runner) run(config *specs.Process) (int, error) {
 	if err != nil {
 		return -1, err
 	}
+	defer func() {
+		// We should terminate the process once we got an error.
+		if retErr != nil {
+			r.terminate(process)
```

Comment:
This is duplicated now. The function deferred previously will also do the exact same thing. Is this on purpose?